inventory.py

```diff
@@ -144,12 +144,22 @@ def pull_enrollment_data_from_udw(course_ids) -> pd.DataFrame:
     return enrollment_df
 
 
+def process_um_id(id: str) -> Union[int, None]:
+    try:
+        um_id = int(id)
+    except ValueError:
+        logger.debug(f'Invalid um_id found: {id}')
+        um_id = None
+    return um_id
+
+
 def pull_user_data_from_udw(user_ids: Sequence[int]) -> pd.DataFrame:
     users_string = ','.join([str(user_id) for user_id in user_ids])
     user_query = f'''
         SELECT u.id AS warehouse_id,
                u.canvas_id AS canvas_id,
                u.name AS name,
+               p.sis_user_id AS um_id,
```

Comment:
Nitpicking sis_id resonates with people more than umid

Reply:
I'm fine changing it, I was just trying to be direct. Anyone else have opinions?

Reply:
It's probably better to keep it less UM specific...

Reply:
can we use sis_id instead?

Reply:
Done!